assets/js/admin/modula-gallery-upload.js

```diff
@@ -60,16 +63,35 @@ class ModulaBrowseForFile {
 				window.send_to_editor = window.send_to_browse_file_url;
 
 				tb_show(
-					modulaGalleryUpload.browseFolder,
+					'<h1>' + modulaGalleryUpload.browseFolder + '</h1>',
 					'media-upload.php?post_id=' +
 						instance.postID +
-						'&amp;type=modula_file_browser&amp;from=wpdlm01&amp;TB_iframe=true'
+						'&amp;type=modula_file_browser&amp;from=wpdlm01&amp;TB_iframe=true&amp;width=800&amp;height=600'				
 				);
 
 				return false;
 			} );
 			instance.readIframeData();
 		}
+		const foldersCheckboxes = document.querySelectorAll(
```

Comment:
aici asta pare super dubios facut - uite un approach mai bun. codu nu-i testat e scris la graba
```suggestion
const modulaFileBrowser = document.querySelector('.modula_file_browser');

if (modulaFileBrowser) {
    modulaFileBrowser.addEventListener('change', function(event) {
        if (event.target.matches('input[type="checkbox"]')) {
            updateSendButtonState();
        }
    });
}

function updateSendButtonState() {
    const checkedCheckboxes = modulaFileBrowser.querySelectorAll('input[type="checkbox"]:checked');
    modulaSendFoldersButton.classList.toggle('disabled', checkedCheckboxes.length === 0);
}

// Initial button state
updateSendButtonState();
```

Reply:
Are commit aici: b59988a123900ab4351e70ad66b521da560ee3e3